### ISSUE TITLE
synStoreMany: creates new versions of files every time  

### ISSUE THREAD

**Issue by amnahsiddiqa** [ArtemSokolov/synExtra#9]
synStoreMany: Everytime I update a directory to synapse using synStoreMany , all of the  data files are updated with new version -even if it is not changed. This is not the normal behavior of Synapse as mentioned @ https://docs.synapse.org/articles/files_and_versioning.html
If the file does not change its md5sum will be identical to the most recent version and hence should not update the version. Why its not like that when I use synSToreMany.

**Comment by ArtemSokolov** [comment]
Hi @amnahsiddiqa,

`synStoreMany()` calls `synStore()` in a recursive fashion. I just added an ability to specify all underlying parameters of `synStore()` directly in `synStoreMany()`. After installing the new version of the package, you should be able to do:

``` r
synStoreMany( "/path/to/data/", "syn123", forceVersion=FALSE )
```

where `forceVersion` is defined in the [synapser API](https://r-docs.synapse.org/reference/synStore.html) as

> forceVersion -- optional named parameter: Indicates whether the method should increment the version of the object even if nothing has changed. Defaults to TRUE.

**Comment by ArtemSokolov** [comment]
Addressed in #10

**Comment by amnahsiddiqa** [comment]
Thanks much!
